commands/createfixpullrequests_test.go

```diff
@@ -116,20 +124,23 @@ func getMavenFixPackageVersionFunc() func(test packageFixTest) CreateFixPullRequ
 func TestFixPackageVersion(t *testing.T) {
 	currentDir, testdataDir := getTestDataDir(t)
 	for _, test := range packageFixTests {
-		// Create temp technology project
-		projectPath := filepath.Join(testdataDir, test.technology.ToString())
-		tmpProjectPath, cleanup := testdatautils.CreateTestProject(t, projectPath)
-		defer cleanup()
-		assert.NoError(t, os.Chdir(tmpProjectPath))
+		func() {
```

Comment:
What is the reson for creating this function here?

Reply:
defer inside a loop